connectivity/check/logging.go

```diff
@@ -128,6 +128,14 @@ func (ct *ConnectivityTest) Fatalf(format string, a ...interface{}) {
 // should always call the methods implemented on Test.
 //
 
+// Progress adds a progress indicator if logging is buffered
+func (t *Test) Progress() {
+	// Print progress if logging is buffered
+	if t.logBuf != nil {
+		t.ctx.params.Writer.Write([]byte{'.'})
```

Comment:
```suggestion
		fmt.Fprint(ct.params.Writer, ".")
```

Reply:
Need to take out a read lock on `logMu` here.

Reply:
Right, added locking.